to.etc.domui/src/main/java/to/etc/domui/component2/lookupinput/LookupInputBase2.java

```diff
@@ -804,13 +804,13 @@ public void setValue(@Nullable OT v) {
 			return;
 		OT old = m_value;
 		m_value = v;
+		getClearButton().setDisplay(DisplayType.INLINE);
```

Comment:
It seems to me like this is not needed as we're not changing the display type anymore.